src/pages/Account/SSHKeys/index.jsx

```diff
@@ -398,23 +380,13 @@ export default class SSHKeys extends Component {
   }
 
   render() {
-    const { keyPairs } = this.props.sshKeyStore;
+    const { sshKeyStore } = this.props;
 
     return (
-      <div>
-        {keyPairs.length ? (
-          this.renderDetail()
-        ) : (
-          <CreateResource
-            title={'Create SSH Key'}
-            aside={this.renderAside()}
-            asideTitle="SSH Keys"
-          >
-            {this.renderForm()}
-          </CreateResource>
-        )}
+      <Loading isLoading={sshKeyStore.isLoading}>
+        {this.renderDetail()}
         {this.renderModals()}
-      </div>
+      </Loading>
```

Comment:
There design is  if no keyPairs data, will show create form in page.

Reply:
ok,  will change it